examples/LoRandom/LoRandom.ino

```diff
@@ -0,0 +1,92 @@
+/*
```

Comment:
Please add this example to the [.travis.yml](https://github.com/sandeepmistry/arduino-LoRa/blob/master/.travis.yml#L39) so it can be compiled as part of the CI.